app/src/main/java/it/niedermann/owncloud/notes/edit/NoteEditFragment.java

```diff
@@ -166,11 +163,28 @@ protected void onNoteLoaded(Note note) {
         }
     }
 
+    private void openSoftKeyboard() {
+        binding.editContent.postDelayed(() -> {
+            binding.editContent.requestFocus();
+
+            final InputMethodManager imm = (InputMethodManager) requireContext().getSystemService(Context.INPUT_METHOD_SERVICE);
+            if (imm != null) {
+                imm.showSoftInput(binding.editContent, InputMethodManager.SHOW_IMPLICIT);
+            } else {
+                Log.e(TAG, InputMethodManager.class.getSimpleName() + " is null.");
+            }
+            //Without a small delay the keyboard does not show reliably
+        }, 100);
+    }
+
     @Override
     public void onPause() {
         super.onPause();
         binding.editContent.removeTextChangedListener(textWatcher);
         cancelTimers();
+
+        final View parentView = ((ViewGroup) requireActivity().findViewById(android.R.id.content)).getChildAt(0);
```

Comment:
This looks a bit riscy in my opinion. As the feature itself is not that important, we should implement it more defensive, add `null` checks and just fallback to `false` if something unexpected happens here. Better be more verbose, the code has already an expiration date (API level 30).